bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Bookie.java

```diff
@@ -1248,7 +1248,7 @@ private LedgerDescriptor getLedgerForEntry(ByteBuf entry, final byte[] masterKey
                 bb.put(masterKey);
                 bb.flip();
 
-                getJournal(ledgerId).logAddEntry(bb, new NopWriteCallback(), null);
+                getJournal(ledgerId).logAddEntry(bb, false /* ackBeforeForce */, new NopWriteCallback(), null);
```

Comment:
we stick to `deferred_sync`. I would suggest us using `ackBeforeSync` rather than `ackBeforeForce`. we should make sure naming consistent across the places.

Reply:
ok for "ackBeforeSync"